ai/lore/lore-ai.js

```diff
@@ -16,6 +16,11 @@ const numGenerateTries = 5;
 const temperature = 1;
 const top_p = 1;
 
+//--------------------------------- ConvAI Mods ------------------------------
+// Initializing the action generation URL from ConvAI
+const actionGenerationURl = `https://test.convai.com/getActions`
```

Comment:
This is not how we integrate apis. All generators must support all modes of operation. We do not special case them.